diff --git a/habitat/utils/visualizations/maps.py b/habitat/utils/visualizations/maps.py
index 31cf9b5fa..fe28f5831 100644
--- a/habitat/utils/visualizations/maps.py
+++ b/habitat/utils/visualizations/maps.py
@@ -43,7 +43,7 @@ MAP_TARGET_BOUNDING_BOX = 9
 TOP_DOWN_MAP_COLORS = np.full((256, 3), 150, dtype=np.uint8)
 TOP_DOWN_MAP_COLORS[10:] = cv2.applyColorMap(
     np.arange(246, dtype=np.uint8), cv2.COLORMAP_JET
-).squeeze(1)[:, ::-1]
+).reshape(-1, 3)[:, ::-1]
 TOP_DOWN_MAP_COLORS[MAP_INVALID_POINT] = [255, 255, 255]  # White
 TOP_DOWN_MAP_COLORS[MAP_VALID_POINT] = [150, 150, 150]  # Light Grey
 TOP_DOWN_MAP_COLORS[MAP_BORDER_INDICATOR] = [50, 50, 50]  # Grey
diff --git a/habitat_baselines/rl/models/rnn_state_encoder.py b/habitat_baselines/rl/models/rnn_state_encoder.py
index f2c8be21e..21d130e87 100644
--- a/habitat_baselines/rl/models/rnn_state_encoder.py
+++ b/habitat_baselines/rl/models/rnn_state_encoder.py
@@ -278,7 +278,7 @@ class RNNStateEncoder(nn.Module):
         return hidden_states
 
     def unpack_hidden(self, hidden_states: torch.Tensor) -> torch.Tensor:
-        return hidden_states
+        return hidden_states.contiguous()
 
     def single_forward(
         self, x, hidden_states, masks
